go/vt/vttablet/onlineddl/executor_test.go

```diff
@@ -74,7 +86,35 @@ func TestValidateAndEditCreateTableStatement(t *testing.T) {
 			expectError:      true,
```

Comment:
IMO it might be worth checking the error message too.

Reply:
done!